Jenkinsfile

```diff
@@ -67,7 +67,8 @@ pipeline {
 					
 					unset JAVA_TOOL_OPTIONS
 					unset _JAVA_OPTIONS
-					mvn install -DskipTests -Djava.io.tmpdir=$WORKSPACE/tmp
+					# force qualifier to start with `z` so we identify it more easily and it always seem more recent than upstrea
+					mvn install -DskipTests -Djava.io.tmpdir=$WORKSPACE/tmp -Dtycho.buildqualifier.format='z'yyyyMMdd-HHmm
```

Comment:
I think you have to actually make it -Dtycho.buildqualifier.format="'z'yyyyMMdd-HHmm" to prevent shell from handling '' and thus having zyyyyMMdd-HHmm passed to java as z will make timezone printed.